src/tests/python-test.py

```diff
@@ -43,14 +45,16 @@ def _run_and_check(self, runme):
     def _get_object_info(self, name, subtree, domain):
         search_dn = "dn=name=%s,cn=%s,cn=%s,cn=sysdb" % (name, subtree, domain)
         try:
-            output = subprocess.check_call("ldbsearch -H %s %s" % (self.local_path,search_dn),
+            output = subprocess.check_call("ldbsearch -H %s %s" %
+                                           (self.local_path, search_dn),
```

Comment:
Have you tried this test? because we do not execute it.

Reply:
I ran our CI, That's the only test done from my part.